<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Meovalu" failures="1" tests="9" name="edgeTest" time="72.595" errors="0" timestamp="2024-12-21T16:10:07 GMT+07:00">
  <testcase classname="testCases.TC001_LoginTest" name="verifyLogin" time="0.717"/>
  <testcase classname="testCases.TC002_AddToCart" name="verifyLogin" time="0.815"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="4.068"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="3.498"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="3.863"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="3.915"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="3.833"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="3.484"/>
  <testcase classname="testCases.TC003_DDT" name="verify_loginDDT" time="0.814">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_DDT.verify_loginDDT(TC003_DDT.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- edgeTest -->
